File: Linguagem C++/Estrutura Enquanto/Crescente.cpp
#include <iostream>

using namespace std;

int main()
{
    int x,y;

    cout << "Digite dois numeros:" << endl;
    cin >> x;
    cin >> y;

    while (x != y){
      if (x > y){
        cout << "DESCRESCENTE" << endl;
      }else {
          cout << "CRESCENTE" << endl;
      };
      cout << "Digite outros dois numeros:" << endl;
      cin >> x;
      cin >> y;
    };

    return 0;
}

File: Linguagem Python/Enquanto/Crescente.py
print("Digite dois numeros:")
x = int(input())
y = int(input())

while x != y:
    if x < y:
        print("CRESCENTE!")
    else:
        print("DESCRESCENTE!")
    print("DIGITE OUTROS DOIS NUMEROS:")
    x = int(input())
    y = int(input())